system-config/99z-qubes-mark-ready.rules

```diff
@@ -0,0 +1,13 @@
+SUBSYSTEM!="block", GOTO="qubes-rule-end"
+ACTION=="remove", GOTO="qubes-rule-end"
+
+# Skip devices that might contain untrusted data.  We won’t be doing any other
+# processing on them (and rightly so).
+KERNEL=="loop*", ATTR{loop/backing_file}!="/var/lib/qubes/vm-kernels/*", ENV{SYSTEMD_READY}="1"
+ENV{DM_LV_NAME}=="vm-*", ENV{SYSTEMD_READY}="1"
+ENV{DM_UUID}=="CRYPT-*", ENV{DM_NAME}!="swap|luks-*", ENV{SYSTEMD_READY}="1"
+ENV{DM_NAME}=="vm-*", ENV{SYSTEMD_READY}="1"
```

Comment:
This belongs to your own custom setup, not anything done by standard qubes code, right? This matches raw device mapper name, not LV name (which is covered earlier) and we don't use dm devices named `vm-*`. Am I missing something? Otherwise, plase remove.